### DISCUSSION TITLE
If idle-timeout function is supported

### DISCUSSION THREAD

**Discussion by ycsong1212** [coder/code-server#5007] [General]
I am working on develop online VSCode support by code-server. To enhance the resource utilization rate, I require cull-idle-timeout mechanism. If users do not use for a period of time, I hope the container killed. According to the doc, code-server provides the heartbeat mechanism. If I keep the WebPage on but do nothing, the time of heartbeat will be updated every minute. This is not what I want. I have seen that such feature is referred like issue #1636 @nhooyr. However, I failed to launch with --idle-timeout args. Is such function supported right now? If not, is there any methods to support my requirement? I'll appreciate it if someone could help.

**Comment by jsjoeio** [comment]
I don't think something like `--idle-timeout` is supported
